### ISSUE TITLE
Feature idea: Add a button to export the data from the simulation into a file.

### ISSUE THREAD

**Issue by PowerMacFan** [s-andrews/virusbreak#48]
I got a idea where there would be a button that would export some stuff that happened in the simulation (eg: virus stats, infection speed, etc.) to a file the user can download. Does anyone else think this is a good idea?

**Comment by s-andrews** [comment]
Do you mean exporting the stuff which is currently recorded in the graph?  If so then sure, I could see that potentially being useful as long as it was fairly unobtrusive and didn't interfere with the main UI.

If you want to have a go then feel free to fork the repository and send me a pull request if you get it working.

**Comment by PowerMacFan** [comment]
Yeah, that would be great! Maybe the button could be next to the graph button?
